src/colorizer/ColorizeCanvas.ts

```diff
@@ -85,7 +89,8 @@ const getDefaultUniforms = (): ColorizeUniforms => {
   const emptyOutliers = packDataTexture([0], FeatureDataType.U8);
   const emptyInRangeIds = packDataTexture([0], FeatureDataType.U8);
   const emptyColorRamp = new ColorRamp(["black"]).texture;
-  const emptyOverlay = new DataTexture(new Uint8Array([0, 0, 0, 0]), 1, 1, RGBAIntegerFormat, UnsignedByteType);
+  const emptyOverlay = new DataTexture(new Uint8Array([0, 0, 0, 0]), 1, 1, RGBAFormat, UnsignedByteType);
+  const emptyBackdrop = new DataTexture(new Uint8Array([1, 0, 0, 0]), 1, 1, RGBAFormat, UnsignedByteType);
```

Comment:
again kind of hard to tell what "overlay" means here. does the overlay mean the main colorized image?

Reply:
I know what would help, is if `emptyFrame` were also right next to the lines for emptyOverlay and emptyBackdrop. :)

Reply:
The overlay is the extra UI overlaid on top of the main view, like the timestamp and scalebar.